tests/test_adders_listed_value_adder.py

```diff
@@ -64,34 +78,60 @@ def test__add_to_inventory_of_listed_values_throws_exception_with_existing_value
             test_adder.add_listed_value(**bad_args)
 
 
-def test__add_to_inventory_of_listed_values_adds_good_value(test_adder):
+def test_add_listed_value_throws_exception_with_empty_args(test_adder):
+    for bad_set_of_values in (
+        {"feature_id": "", "new_value_en": "Value", "new_value_ru": "Значение"},
+        {"feature_id": "A-1", "new_value_en": "", "new_value_ru": "Значение"},
+        {"feature_id": "A-1", "new_value_en": "Value", "new_value_ru": ""},
+    ):
+        with pytest.raises(ListedValueAdderError, match="None of the passed strings can be empty"):
+            test_adder.add_listed_value(**bad_set_of_values)
+
+
+def test__add_after_value_that_exceeds_maximum(test_adder):
+    with pytest.raises(ListedValueAdderError, match="must be between 1 and 5"):
+        test_adder.add_listed_value(
+            feature_id="A-2",
+            new_value_en="something",
+            new_value_ru="что-нибудь",
+            index_to_assign=16,
+        )
```

Comment:
we have to check cases that not only exceed maximum, but also `0` or `-2`.
the following cases should be checked if, say, feature `A-2` has values from `A-2-1` to `A-2-15`
- `-2` (edge case, just one less than `-1`)
- `0` (edge case, just one less than `1`)
- `16` (edge case, just one more than last index)
- `100` (normal test case, should obviously fail, test it just in case)

We can include more test cases here than we actually need because checking as many numbers as we want is super easy and does not require any additional coding.

Reply:
Done! Missing tests are now added and also I reordered them according to the arguments order